sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosChangeFeedRequestOptions.java

```diff
@@ -87,8 +85,6 @@ ChangeFeedState getContinuation() {
      *
      * @return the feed range.
      */
-    @Beta(value = Beta.SinceVersion.V4_12_0, warningText =
-        Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public FeedRange getFeedRange() {
```

Comment:
I thought that FeedRange still had some work to be done (may be in regards to merge support) before we GA it. 
@FabianMeiswinkel  @xinlian12  ?

Reply:
Nothing that could break these APIs - we have sipped it too long ago and are using for a while - so breaking it isn't an option IMO